include/file_process.hpp

```diff
@@ -0,0 +1,49 @@
+/** \brief This header file declares functions for Output File Processing.
+*
+* The output file generated by the simulator is processed
+* to produce a more readable format. The data is stored in 
+* output file *_proc.txt in a table format as shown below:
+*
+*Time           value  port                component
+*00:00:10:000   20     out                 generator_con  
+*00:00:20:000   1      packet_sent_out     sender1        
+*00:00:20:000   11     data_out            sender1        
+*00:00:23:000   11     out                 subnet1        
+*00:00:33:000   1      out                 receiver1      
+*00:00:36:000   1      out                 subnet2  
+*
+* Peter Bliznyuk-Kvitko
+* Carleton University
+*
+*/
+
+#ifndef _FILE_PROCESS_HPP_
```

Comment:
Headers guards would have double underscore in the beginning and end i.e., __FILE_PROCESS_HPP__
I request you to please correct it.

Reply:
Solved